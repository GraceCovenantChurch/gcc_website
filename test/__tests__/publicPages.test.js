/* eslint-env node, jest */

import request from 'supertest-as-promised';
import pretty from 'pretty';

import { withPublicServer } from '../serverSetup';

describe('Public ServerTest', () => {
  let server = withPublicServer();

  it('renders Home page', () =>
    request(server())
      .get('/')
      .expect(200)
      .then(res => expect(pretty(res.text)).toMatchSnapshot()));

  it('renders Welcome page', () =>
    request(server())
      .get('/welcome')
      .expect(200)
      .then(res => expect(pretty(res.text)).toMatchSnapshot()));

  it('renders Family Group page', () =>
    request(server())
      .get('/familygroup')
      .expect(200)
      .then(res => expect(pretty(res.text)).toMatchSnapshot()));

  it('renders 404 page', () =>
    request(server())
      .get('/invalid')
      .expect(200)
      .then(res => expect(pretty(res.text)).toMatchSnapshot()));
});
