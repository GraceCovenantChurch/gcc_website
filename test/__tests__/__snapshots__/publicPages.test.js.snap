// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Public ServerTest renders 404 page 1`] = `
"<!doctype html>
<html>

  <head>
    <title data-react-helmet=\\"true\\">Error: Not Found - Grace Covenant Church</title>

    <link data-react-helmet=\\"true\\" href=\\"https://fonts.googleapis.com/css?family=Roboto:300,700\\" rel=\\"stylesheet\\" />
    <link href=\\"https://fonts.googleapis.com/css?family=PT+Sans\\" rel=\\"stylesheet\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/app.bundle.css\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/public.bundle.css\\" />
  </head>

  <body>
    <div id=\\"app\\">
      <div>
        <nav id=\\"Navbar__mainNavbar\\" class=\\"navbar navbar-expand-lg navbar-light Navbar__navbar-light\\">
          <a class=\\"navbar-brand Navbar__navbar-brand\\" href=\\"/\\">
            <h1 class=\\"sr-only\\">Grace Covenant Church</h1>
            <img class=\\"Navbar__logo-light\\" src=\\"/static/images/gcclogo.png\\" alt=\\"Grace Covenant Church\\" />
            <img class=\\"Navbar__logo-dark\\" src=\\"/static/images/gcclogo-black.png\\" alt=\\"Grace Covenant Church\\" />
          </a>
          <button class=\\"navbar-toggler Navbar__navbar-toggler\\" type=\\"button\\"><span class=\\"navbar-toggler-icon Navbar__navbar-toggler-icon\\"></span></button>
          <div class=\\"navbar-collapse collapse\\" style=\\"height:auto\\">
            <ul class=\\"navbar-nav ml-auto Navbar__navbar-nav\\">
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/welcome\\">I&#x27;m New</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">About</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/familygroup\\">Family Groups</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/ministries\\">Ministries</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/page\\">Giving</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Multimedia</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Events</a>
              </li>
            </ul>
          </div>
        </nav>
        <main>
          <div>Nothing here!</div>
        </main>
        <footer>
          <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
            <div class=\\"Center__centered\\">
              <div class=\\"container footer\\">
                <div class=\\"Footer__icons\\">
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.facebook.com/gracecovenant/\\">
                    <i class=\\"fa fa-facebook fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://twitter.com/gccphiladelphia\\">
                    <i class=\\"fa fa-twitter fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.instagram.com/gccphiladelphia/\\">
                    <i class=\\"fa fa-instagram fa-2x\\"></i>
                  </a>
                </div>
                <p>Â© Grace Covenant Church</p>
              </div>
            </div>
          </div>
        </footer>
      </div>
    </div>
    <script type=\\"text/javascript\\">
      window.__INITIAL_STATE__ = {
        \\"metadata\\": {
          \\"title\\": \\"Error: Not Found - Grace Covenant Church\\"
        },
        \\"modelData\\": {},
        \\"page\\": {
          \\"title\\": \\"\\",
          \\"content\\": \\"\\"
        }
      }
    </script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/manifest.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/react.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/public.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/app.js\\"></script>
  </body>

</html>"
`;

exports[`Public ServerTest renders Family Group page 1`] = `
"<!doctype html>
<html>

  <head>
    <title data-react-helmet=\\"true\\">Family Group - Grace Covenant Church</title>

    <link data-react-helmet=\\"true\\" href=\\"https://fonts.googleapis.com/css?family=Roboto:300,700\\" rel=\\"stylesheet\\" />
    <link data-react-helmet=\\"true\\" rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/pages/FamilyGroup.bundle.css\\" />
    <link href=\\"https://fonts.googleapis.com/css?family=PT+Sans\\" rel=\\"stylesheet\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/app.bundle.css\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/public.bundle.css\\" />
  </head>

  <body>
    <div id=\\"app\\">
      <div>
        <nav id=\\"Navbar__mainNavbar\\" class=\\"navbar navbar-expand-lg navbar-light Navbar__navbar-light\\">
          <a class=\\"navbar-brand Navbar__navbar-brand\\" href=\\"/\\">
            <h1 class=\\"sr-only\\">Grace Covenant Church</h1>
            <img class=\\"Navbar__logo-light\\" src=\\"/static/images/gcclogo.png\\" alt=\\"Grace Covenant Church\\" />
            <img class=\\"Navbar__logo-dark\\" src=\\"/static/images/gcclogo-black.png\\" alt=\\"Grace Covenant Church\\" />
          </a>
          <button class=\\"navbar-toggler Navbar__navbar-toggler\\" type=\\"button\\"><span class=\\"navbar-toggler-icon Navbar__navbar-toggler-icon\\"></span></button>
          <div class=\\"navbar-collapse collapse\\" style=\\"height:auto\\">
            <ul class=\\"navbar-nav ml-auto Navbar__navbar-nav\\">
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/welcome\\">I&#x27;m New</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">About</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/familygroup\\">Family Groups</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/ministries\\">Ministries</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/page\\">Giving</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Multimedia</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Events</a>
              </li>
            </ul>
          </div>
        </nav>
        <main>
          <div>
            <div class=\\"jumbotron Jumbotron__jumbotron\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(/static/images/familygroup/fg_background.jpg);background-attachment:local;background-position:center;background-size:cover\\"></div>
              </div>
              <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
                <div class=\\"Center__centered\\">
                  <div class=\\"TitleBanner__titleText MainTemplate__header\\" style=\\"color:white\\">Family Group</div>
                </div>
              </div>
            </div>
            <div class=\\"FamilyGroup__pageContent\\">
              <div class=\\"description\\">At GCC we place a high emphasis on Family Groups because we believe that it is in these smaller settings that we are able to meet with God, building meaningful relationships, and develop biblical accountability. Family groups meet on a weekly
                basis at various places.
                <br/>
                <br/>Through Bible study, discussions, sharing and prayer, we learn to live out the teachings of the Bible in our daily lives. You are strongly encouraged to join a family group that fits your needs,interests and life situation.
                <br/>
                <br/>Family groups are divided between the Harvest (undergraduate), Crossroad (post-college), and International groups.
                <br/>
                <br/>Please watch the video below to learn more!</div>
              <iframe class=\\"FamilyGroup__video\\" src=\\"https://www.youtube.com/embed/rvXubANV7RA\\" frameBorder=\\"0\\" title=\\"intro\\" allow=\\"autoplay; encrypted-media\\" allowfullscreen=\\"\\"></iframe>
              <div class=\\"FamilyGroup__signupTable\\">
                <div class=\\"FamilyGroup__tableRow\\">
                  <div class=\\"FamilyGroup__rowTitle\\">College</div>
                  <div class=\\"FamilyGroup__rowDays\\">Mondays to Thursdays</div>
                  <div class=\\"FamilyGroup__rowDescription\\">Our College Family Groups meet once a week on Drexel, Moore, Penn, Temple, and USciences!</div>
                  <div class=\\"FamilyGroup__rowSignup\\">
                    <a href=\\"#\\">Sign Up &gt;</a>
                  </div>
                </div>
                <div class=\\"FamilyGroup__tableRow\\">
                  <div class=\\"FamilyGroup__rowTitle\\">Young Adults</div>
                  <div class=\\"FamilyGroup__rowDays\\">Mondays to Thursdays</div>
                  <div class=\\"FamilyGroup__rowDescription\\">Our Young Adult Family Groups have a mix of post-undergraduate, grad students, PhD&#x27;s, working adults, and young married couples.</div>
                  <div class=\\"FamilyGroup__rowSignup\\">
                    <a href=\\"#\\">Sign Up &gt;</a>
                  </div>
                </div>
                <div class=\\"FamilyGroup__tableRow\\">
                  <div class=\\"FamilyGroup__rowTitle\\">International</div>
                  <div class=\\"FamilyGroup__rowDays\\">Thursdays</div>
                  <div class=\\"FamilyGroup__rowDescription\\">Description needed</div>
                  <div class=\\"FamilyGroup__rowSignup\\">
                    <a href=\\"#\\">Sign Up &gt;</a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </main>
        <footer>
          <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
            <div class=\\"Center__centered\\">
              <div class=\\"container footer\\">
                <div class=\\"Footer__icons\\">
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.facebook.com/gracecovenant/\\">
                    <i class=\\"fa fa-facebook fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://twitter.com/gccphiladelphia\\">
                    <i class=\\"fa fa-twitter fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.instagram.com/gccphiladelphia/\\">
                    <i class=\\"fa fa-instagram fa-2x\\"></i>
                  </a>
                </div>
                <p>Â© Grace Covenant Church</p>
              </div>
            </div>
          </div>
        </footer>
      </div>
    </div>
    <script type=\\"text/javascript\\">
      window.__INITIAL_STATE__ = {
        \\"metadata\\": {
          \\"title\\": \\"Family Group - Grace Covenant Church\\"
        },
        \\"modelData\\": {},
        \\"page\\": {
          \\"title\\": \\"\\",
          \\"content\\": \\"\\"
        }
      }
    </script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/manifest.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/react.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/public.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/app.js\\"></script>
  </body>

</html>"
`;

exports[`Public ServerTest renders Home page 1`] = `
"<!doctype html>
<html>

  <head>
    <title data-react-helmet=\\"true\\">Grace Covenant Church</title>

    <link data-react-helmet=\\"true\\" href=\\"https://fonts.googleapis.com/css?family=Roboto:300,700\\" rel=\\"stylesheet\\" />
    <link data-react-helmet=\\"true\\" rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/pages/Home.bundle.css\\" />
    <link href=\\"https://fonts.googleapis.com/css?family=PT+Sans\\" rel=\\"stylesheet\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/app.bundle.css\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/public.bundle.css\\" />
  </head>

  <body>
    <div id=\\"app\\">
      <div>
        <nav id=\\"Navbar__mainNavbar\\" class=\\"navbar navbar-expand-lg navbar-light Navbar__navbar-light\\">
          <a class=\\"navbar-brand Navbar__navbar-brand\\" href=\\"/\\">
            <h1 class=\\"sr-only\\">Grace Covenant Church</h1>
            <img class=\\"Navbar__logo-light\\" src=\\"/static/images/gcclogo.png\\" alt=\\"Grace Covenant Church\\" />
            <img class=\\"Navbar__logo-dark\\" src=\\"/static/images/gcclogo-black.png\\" alt=\\"Grace Covenant Church\\" />
          </a>
          <button class=\\"navbar-toggler Navbar__navbar-toggler\\" type=\\"button\\"><span class=\\"navbar-toggler-icon Navbar__navbar-toggler-icon\\"></span></button>
          <div class=\\"navbar-collapse collapse\\" style=\\"height:auto\\">
            <ul class=\\"navbar-nav ml-auto Navbar__navbar-nav\\">
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/welcome\\">I&#x27;m New</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">About</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/familygroup\\">Family Groups</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/ministries\\">Ministries</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/page\\">Giving</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Multimedia</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Events</a>
              </li>
            </ul>
          </div>
        </nav>
        <main>
          <div id=\\"home\\">
            <div class=\\"jumbotron Jumbotron__jumbotron\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(/static/images/home/philly.jpg);background-attachment:local;background-position:top left;background-size:cover\\"></div>
              </div>
              <div class=\\"Home__title\\" style=\\"color:white;top:40%\\">
                <h1 class=\\"MainTemplate__header\\">Grace Covenant Church</h1>
                <div class=\\"Home__subtitle\\">
                  <h3>Raising up Kingdom workers who are
                    <br/>influenced by Christ to change the world.</h3>
                </div>
              </div>
            </div>
            <div class=\\"jumbotron Jumbotron__jumbotron\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(/static/images/home/welcome.jpg);background-attachment:local;background-position:top left;background-size:cover\\"></div>
              </div>
              <div class=\\"Banner__title\\" style=\\"color:white;top:30%\\">
                <h1 class=\\"MainTemplate__header\\">Join Us In Worship</h1>
                <div class=\\"Home__subtitle\\">
                  <h3>Sunday Service 11:15AM
                    <br/>Friday Night Live 7:30PM</h3>
                  <h4>Meyerson Hall B-1, 210 South 34th Street
                    <br/>Philadelphia, PA 19104</h4>
                  <a href=\\"/welcome\\">
                    <button class=\\"Home__infoButton\\">More Info</button>
                  </a>
                </div>
              </div>
            </div>
            <div class=\\"jumbotron Jumbotron__jumbotron\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(/static/images/home/familygroup.jpg);background-attachment:local;background-position:top left;background-size:cover\\"></div>
              </div>
              <div class=\\"Banner__title\\" style=\\"color:white;top:30%\\">
                <h1 class=\\"MainTemplate__header\\">Family Group</h1>
                <div class=\\"Home__subtitle\\">
                  <h3>You haven&amp;t checked out GCC if you haven&amp;t checked out our family groups.</h3>
                  <a href=\\"/familygroup\\">
                    <button class=\\"Home__infoButton\\">Sign Up Here</button>
                  </a>
                </div>
              </div>
            </div>
            <div class=\\"jumbotron Jumbotron__jumbotron\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(/static/images/home/events.jpg);background-attachment:local;background-position:top left;background-size:cover\\"></div>
              </div>
              <div class=\\"Banner__title\\" style=\\"color:white;top:20%\\">
                <h1 class=\\"MainTemplate__header\\">Events</h1>
                <div class=\\"Home__subtitle\\"></div>
                <div class=\\"subtitle\\">
                  <a href=\\"/events\\">
                    <button class=\\"Home__infoButton\\">More Info</button>
                  </a>
                </div>
              </div>
            </div>
            <div class=\\"bibleVerse\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(null);background-attachment:fixed;background-position:top left;background-size:cover\\"></div>
              </div>
              <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
                <div class=\\"Center__centered\\">
                  <div class=\\"BannerBibleVerse__bibleVerse\\" style=\\"color:black\\">
                    <h1>February</h1>
                    <h3>Whom have I in heaven but you? And there is nothing on earth that I desire besides you. My flesh and my heart may fail, but God is the strength of my heart and my portion forever.</h3>
                    <h4>Psalm 73:25-26</h4>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </main>
        <footer>
          <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
            <div class=\\"Center__centered\\">
              <div class=\\"container footer\\">
                <div class=\\"Footer__icons\\">
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.facebook.com/gracecovenant/\\">
                    <i class=\\"fa fa-facebook fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://twitter.com/gccphiladelphia\\">
                    <i class=\\"fa fa-twitter fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.instagram.com/gccphiladelphia/\\">
                    <i class=\\"fa fa-instagram fa-2x\\"></i>
                  </a>
                </div>
                <p>Â© Grace Covenant Church</p>
              </div>
            </div>
          </div>
        </footer>
      </div>
    </div>
    <script type=\\"text/javascript\\">
      window.__INITIAL_STATE__ = {
        \\"metadata\\": {
          \\"title\\": \\"Grace Covenant Church\\"
        },
        \\"modelData\\": {},
        \\"page\\": {
          \\"title\\": \\"\\",
          \\"content\\": \\"\\"
        }
      }
    </script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/manifest.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/react.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/public.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/app.js\\"></script>
  </body>

</html>"
`;

exports[`Public ServerTest renders Welcome page 1`] = `
"<!doctype html>
<html>

  <head>
    <title data-react-helmet=\\"true\\">Grace Covenant Church</title>

    <link data-react-helmet=\\"true\\" href=\\"https://fonts.googleapis.com/css?family=Roboto:300,700\\" rel=\\"stylesheet\\" />
    <link data-react-helmet=\\"true\\" rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"public/assets/pages/welcome/Welcome.bundle.css\\" />
    <link href=\\"https://fonts.googleapis.com/css?family=PT+Sans\\" rel=\\"stylesheet\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/app.bundle.css\\" />
    <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/assets/public.bundle.css\\" />
  </head>

  <body>
    <div id=\\"app\\">
      <div>
        <nav id=\\"Navbar__mainNavbar\\" class=\\"navbar navbar-expand-lg navbar-light Navbar__navbar-light\\">
          <a class=\\"navbar-brand Navbar__navbar-brand\\" href=\\"/\\">
            <h1 class=\\"sr-only\\">Grace Covenant Church</h1>
            <img class=\\"Navbar__logo-light\\" src=\\"/static/images/gcclogo.png\\" alt=\\"Grace Covenant Church\\" />
            <img class=\\"Navbar__logo-dark\\" src=\\"/static/images/gcclogo-black.png\\" alt=\\"Grace Covenant Church\\" />
          </a>
          <button class=\\"navbar-toggler Navbar__navbar-toggler\\" type=\\"button\\"><span class=\\"navbar-toggler-icon Navbar__navbar-toggler-icon\\"></span></button>
          <div class=\\"navbar-collapse collapse\\" style=\\"height:auto\\">
            <ul class=\\"navbar-nav ml-auto Navbar__navbar-nav\\">
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/welcome\\">I&#x27;m New</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">About</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/familygroup\\">Family Groups</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/ministries\\">Ministries</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/page\\">Giving</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Multimedia</a>
              </li>
              <li class=\\"nav-item\\">
                <a class=\\"nav-link Navbar__nav-link\\" href=\\"/pages\\">Events</a>
              </li>
            </ul>
          </div>
        </nav>
        <main>
          <div>
            <div class=\\"jumbotron Jumbotron__jumbotron\\" style=\\"height:100vh\\">
              <div class=\\"BackgroundImage__backgroundImageContainer\\">
                <div class=\\"BackgroundImage__backgroundImage\\" style=\\"background-image:url(/static/images/welcome/welcome_background.jpg);background-attachment:local;background-position:center;background-size:cover\\"></div>
              </div>
              <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
                <div class=\\"Center__centered\\">
                  <div class=\\"TitleBanner__titleText MainTemplate__header\\" style=\\"color:white\\">
                    <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
                      <div class=\\"Center__centered\\">Welcome!</div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div class=\\"Welcome__pageContent\\">
              <div class=\\"description\\">We&#x27;re glad you&#x27;re exploring our University City Site! As you take a look around, we hope you catch a glimpse of what God is doing at GCC University City.
                <br/>
                <br/>If you like videos, check out our intro video:</div>
              <div class=\\"Welcome__video\\">
                <iframe width=\\"100%\\" height=\\"100%\\" title=\\"Grace Covenant Church Intro Video\\" src=\\"https://www.youtube.com/embed/5M8H4xhSPWc\\" frameBorder=\\"0\\" allowfullscreen=\\"\\"></iframe>
              </div>
              <div class=\\"description\\">Our doors are wide open to people from all backgrounds. Whether you are a spiritual seeker just beginning to ask questions about God or a committed Christian wanting to deepen your faith, Grace Covenant Church is a place you can call home
                and find spiritual help, hope, and encouragement. Wherever you are in your spiritual journey, we invite you to discover GCCâand discover for yourself Godâs wonderful love and plan for your life!
                <br/>
                <br/>What makes us unique? It isnât our beliefs. We affirm the same basic Christian doctrines that have been a part of the Church for hundreds of years. Rather, we are committed to presenting these truths in a practical, relevant, contemporary,
                and loving manner. Our desire is to introduce you to Jesus Christ and to help you grow and become a kingdom worker. We want to help you to become a kingdom worker who can influence the world around you with the love of Jesus Christ.
                <br/>
                <br/>Our church is comprised of a network of âfamily groupsâ; small group settings where honest friendships, Bible study, constant prayer, and compassionate care are emphasized. It is within these dynamic family groups where the core of our
                evangelism and discipleship take place. We invite you to become of a part of these contagious Christian communities that strive to influence and encourage one another.
                <br/>
                <br/>Again, whether youâre a spiritual seeker whoâs just starting to ask questions about God, or a committed Christian who wants to sink the roots of your faith even deeper, you can find a home here at Grace Covenant Church.
                <br/>
                <br/>Our Sunday Services and Friday Night Large Groups are typically located at Meyerson Hall Room B-1, 3401 Walnut Street Philadelphia, PA 19104. We oftentimes meet at Ralston House, 3615 Chestnut Street Philadelphia, PA 19104, as well. Keep
                an eye out for Grace Covenant Church signs and our welcomers to find us!</div>
              <div class=\\"Welcome__maps\\">
                <div class=\\"row\\">
                  <div class=\\"col-sm-6\\">
                    <div style=\\"width:100%;position:relative;padding-top:75%\\">
                      <div style=\\"position:absolute;top:0;left:0;bottom:0;right:0\\">
                        <iframe frameBorder=\\"0\\" title=\\"Meyerson Hall\\" src=\\"https://www.google.com/maps/embed/v1/place?q=Meyerson+Hall,+Philadelphia,+PA,+United+States&amp;key=GOOGLE_MAPS_KEY\\" style=\\"width:100%;height:100%\\"></iframe>
                      </div>
                    </div>
                  </div>
                  <div class=\\"col-sm-6\\">
                    <div style=\\"width:100%;position:relative;padding-top:75%\\">
                      <div style=\\"position:absolute;top:0;left:0;bottom:0;right:0\\">
                        <iframe frameBorder=\\"0\\" title=\\"Ralston House\\" src=\\"https://www.google.com/maps/embed/v1/place?q=Ralston+House,+Chestnut+Street,+Philadelphia,+PA,+United+States&amp;key=GOOGLE_MAPS_KEY\\" style=\\"width:100%;height:100%\\"></iframe>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div class=\\"description\\">If you would like more information about the church, please continue looking through this website or contact us at web@gracecovenant.net.
                <br/>
                <br/>
              </div>
            </div>
          </div>
        </main>
        <footer>
          <div class=\\"Center__center Center__centerVertical Center__centerHorizontal\\">
            <div class=\\"Center__centered\\">
              <div class=\\"container footer\\">
                <div class=\\"Footer__icons\\">
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.facebook.com/gracecovenant/\\">
                    <i class=\\"fa fa-facebook fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://twitter.com/gccphiladelphia\\">
                    <i class=\\"fa fa-twitter fa-2x\\"></i>
                  </a>
                  <a target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"Footer__footerLinks\\" href=\\"https://www.instagram.com/gccphiladelphia/\\">
                    <i class=\\"fa fa-instagram fa-2x\\"></i>
                  </a>
                </div>
                <p>Â© Grace Covenant Church</p>
              </div>
            </div>
          </div>
        </footer>
      </div>
    </div>
    <script type=\\"text/javascript\\">
      window.__INITIAL_STATE__ = {
        \\"metadata\\": {
          \\"title\\": \\"Grace Covenant Church\\"
        },
        \\"modelData\\": {},
        \\"page\\": {
          \\"title\\": \\"\\",
          \\"content\\": \\"\\"
        }
      }
    </script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/manifest.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/react.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/public.js\\"></script>
    <script type=\\"text/javascript\\" src=\\"/public/assets/app.js\\"></script>
  </body>

</html>"
`;
